### DISCUSSION TITLE
root filesystem extraction failed

### DISCUSSION THREAD

**Discussion by suhuanhou** [sylabs/singularity#1744] [Q&A]
Hey! I'm using singularity with version 3.7.1 in order to use Demuxify.sif.

I install with conda environment:
conda install -c conda-forge singularity

the code problem is:
$ singularity exec Demuxafy.sif Combine_Results.R -h
INFO: Converting SIF file to temporary sandbox...
FATAL: while extracting Demuxafy.sif: root filesystem extraction failed: extract command failed: ERROR : Failed to create user namespace: user namespace disabled
: exit status 1

Thank youu in advance!

Btw，the singularity was installed without root (sudo) and run in the HPC

**Comment by dtrudg** [comment]
This error would be seen because unprivileged user namespaces are not enabled on the system. Please see the documentation:

https://docs.sylabs.io/guides/latest/admin-guide/user_namespace.html#user-namespace-requirements

Is the system running RHEL / CentOS 7? You will probably have to request that unprivileged user namespaces are enabled by the administrator.

Also note that v3.7.1 is very old, and many bugs are fixed in newer versions. The current version of SingularityCE is 3.11.3.
